src/test/scala/io/xskipper/api/util/APITestUtils.scala

```diff
@@ -21,12 +21,34 @@ object APITestUtils {
 
   def parseAPITestDescriptors(fileName: String, inputLocationPrefix: String
   = getDefaultInputLocationPrefix()): Seq[APITestDescriptor] = {
-    val input = scala.io.Source.fromFile(fileName).mkString
+    val input = readTextFileAsString(fileName)
     val parser = new JSONParser()
     val jsonObject = parser.parse(input).asInstanceOf[JSONObject]
     parseAPITestDescriptors(jsonObject, inputLocationPrefix)
   }
 
+  def parseAPITestDescriptors(json: JSONObject,
```

Comment:
why this test was moved?

Reply:
@guykhazma i reformatted the file and it changed its order.
(btw it's not a test, it's a util function to parse the test descriptors from JSON)